dagql/idtui/frontend_plain.go

```diff
@@ -246,7 +243,9 @@ func (fe plainLogExporter) Export(ctx context.Context, logs []sdklog.Record) err
 	fe.mu.Lock()
 	defer fe.mu.Unlock()
 
-	slog.Debug("frontend exporting logs", "logs", len(logs))
+	// NOTE: if we log in here we'll just go into a loop, so...
+	// let's just hope everything is working by now :D
+	// slog.Debug("frontend exporting logs", "logs", len(logs))
```

Comment:
Mm, I have not actually found these logs of *particular* value - do we actually need any of these in `ExportSpans`/etc? They're quite noisy when enabled, and I'm not sure if there's a case where they're useful.

If we want to see the logs that the client receives, then it feels like maybe we should have that in the db?

Reply:
As someone who spent the last month debugging OpenTelemetry draining issues I can attest to their value. :sweat_smile:

Could maybe move them to `ExtraDebug` level but every time we do this it only makes debugging OpenTelemetry more difficult for me. I would suggest just leaving these as-is until the coast feels clear.

> If we want to see the logs that the client receives, then it feels like maybe we should have that in the db?

Not sure what you mean here.

Reply:
Yeah fair enough - worth keeping then!

> Not sure what you mean here.

I mean, we could move them into `ExportSpans` in `db.go`, that way they can be shared between the different frontends, instead. Just a suggestion, it may not work for logistical reasons :sweat_smile: